content/automate/ManageIQ/System/CommonMethods/MiqAe.class/__class__.yaml

```diff
@@ -0,0 +1,53 @@
+---
+object_type: class
```

Comment:
@fdupont-redhat 
This class already exists in master, are there any changes to it.

Reply:
Nope. But apparently, Git think there is...

Reply:
@fdupont-redhat 
When the first PR was created the directory for MiqAe didn't have the class prefix.
The actual name should have been MiqAe.class but it got stored in the content repo as MiqAe.
I am not sure how you created the class, if you look at other class directories under

https://github.com/ManageIQ/manageiq-content/tree/master/content/automate/ManageIQ/System/CommonMethods

You will see that MiqAe stands out.
Can we fix this issue in a separate PR which might help reduce the # of files in this PR?